packages/oauth/src/index.ts

```diff
@@ -91,7 +91,7 @@ export class InstallProvider {
   }
 
   /**
-   * Fetches data from the installationStore for non Org Installations.
```

Comment:
Nice improvements! As this pull request takes a bit long before merging into main branch, can you submit another pull request including some token rotation unrelated changes?

Reply:
Thanks! The correction to the comment above is the only non-token rotation change included here. Since it's not time-sensitive and it's just a comment, I think it's fine to just wait until this PR goes in to see it fixed.